src/updateWeather.js

```diff
@@ -9,23 +10,36 @@ import getWeatherDetails from "./apis/weatherDetails.js";
  *
  */
 export default async (lat, lon) => {
-  const cityEl = document.getElementById("city");
-  const tempEl = document.getElementById("temp");
-  const iconEl = document.getElementById("icon");
+  const $city = $("#city");
+  const $temperature = $("#temp");
+  const $icon = $("#icon");
+  const $humidity = $("#humidity");
+  const $wind = $("#wind");
+  const $pressure = $("#pressure");
+  const $sun = $("#sunrise-sunset");
+  const $feelsLike = $("#feels-like");
+  const $visibility = $("#visibility");
 
   const weatherDetails = await getWeatherDetails(lat, lon);
-  const weather = weatherDetails.weather[0];
 
-  console.log(weatherDetails);
+  const { country, sunset, sunrise } = weatherDetails.sys;
+  const { name, dt, visibility } = weatherDetails;
+  const { feels_like, humidity, pressure, temp, temp_max, temp_min } =
+    weatherDetails.main;
+  const { icon, description } = weatherDetails.weather[0];
+  const { speed, deg } = weatherDetails.wind;
 
-  const country = weatherDetails.sys.country;
-  const cityName = weatherDetails.name;
-  const temp = weatherDetails.main.temp;
-  const iconUrl = weather.icon;
-
-  cityEl.innerHTML = `${cityName}, ${country}`;
-  tempEl.innerHTML = `${temp} &#176;C`;
-  iconEl.innerHTML = `
-        <img src=${iconUrl} alt="${weather.description}" width=120 height=120 />
+  $visibility.innerHTML = `${Math.floor(visibility / 1000)} km`;
```

Comment:
My first thought when I dealing with rounding like that - I usually go with `Math.round()`. Any specific reason for using `Math.floor()` here?